README.md

```diff
@@ -23,7 +23,7 @@ Then, start the program by running:
 $ docker run --rm -v $PWD:/app -p 8000:8000 simphony-docs
 ```
 
-### Linux
+#### Linux
 At an OS level (these commands work on Linux Debian):
 ```shell
 $ sudo apt install pandoc graphviz
```

Comment:
default-jre is missing. Btw the UML rendering is still failing. Am I doing something wrong on my side?

Reply:
I added the jre.
But the rendering is working alright for me